src/e2e-test/features/successfactors/DesignTimeValidation.feature

```diff
@@ -0,0 +1,92 @@
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+
+@SuccessFactorsSource
+@Smoke
+@Regression
+Feature: SuccessFactors Source - Design time validation scenarios
+
+  @BATCH-TS-SCFA-DSGN-ERROR-01
+  Scenario: Verify required fields missing validation for listed properties
+    When Open Datafusion Project to configure pipeline
+    And Select data pipeline type as: "Batch"
+    And Select plugin: "SAP SuccessFactors" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "SAP SuccessFactors"
+    And Click on the Validate button
+    Then Verify mandatory property error for below listed properties:
+      | referenceName    |
+      | baseURL          |
+      | entityName       |
+      | username         |
+      | password         |
+
+
+  @BATCH-TS-SCFA-DSGN-ERROR-02
+  Scenario: Verify validation message when user provide invalid BaseURL
```

Comment:
*provides an

Reply:
Apply similar changes in other places as well

Reply:
updated